docs/applications.md

```diff
@@ -45,10 +45,19 @@ You can use any of the following to add handled routes to the application:
 
 * `app.add_route(path, func, methods=["GET"])` - Add an HTTP route. The function may be either a coroutine or a regular function, with a signature like `func(request, **kwargs) -> response`.
 * `app.add_websocket_route(path, func)` - Add a websocket session route. The function must be a coroutine, with a signature like `func(session, **kwargs)`.
-* `app.add_graphql_route(path, schema, executor=None) - Add a GraphQL route.
+* `app.add_graphql_route(path, schema, executor=None)` - Add a GraphQL route.
 * `@app.route(path)` - Add an HTTP route, decorator style.
 * `@app.websocket_route(path)` - Add a WebSocket route, decorator style.
 
+### Adding event handlers to the application
+
+There are two ways to add lifespan events:
```

Comment:
Just "to add event handlers" I think. We'll have other things here soonish... (timer ticks and wall clock events)